### PR TITLE
Art gallery asset and music changes

### PR DESCRIPTION
**Pull request by Oliveriver**
**Brief Description of What This PR Does**

* Replaced jazz theme with the updated version.
* Added title and artist info for all music tracks in the gallery.
* Added title and artist info for all concept art in the gallery.
* Added more concept art.
* Alphabetised concept art list (within each category).

Note that I've added ~sciocont's concept art since we recently received written permission to use his assets. As far as I know, he hasn't yet signed the CLA, so I can nudge him again if we want to be certain about it.

**Related Issues**

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
> Note that I've added ~sciocont's concept art since we recently received written permission to use his assets. As far as I know, he hasn't yet signed the CLA, so I can nudge him again if we want to be certain about it.

I don't see that as absolutely necessary but would be a nice extra guarantee. I think we are pretty safe if that work is used only as a single art piece in the game, which would be easy to remove if there are problems or something.

File: assets/sounds/main-menu-theme-jazz.ogg.import [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+path="res://.import/main-menu-theme-jazz.ogg-0c52f9a3a6f0539ffbbc6848c019a979.oggstr"
+
+[deps]
+
+source_file="res://assets/sounds/main-menu-theme-jazz.ogg"
+dest_files=[ "res://.import/main-menu-theme-jazz.ogg-0c52f9a3a6f0539ffbbc6848c019a979.oggstr" ]
+
+[params]
+
+loop=true
```
Comment (hhyyrylainen) [inline assets/sounds/main-menu-theme-jazz.ogg.import:14]: Loop needs to be set to false.

**Review by hhyyrylainen — APPROVED** [review]
Fixed the loop property and looked through the functionality in general to check it's good. I did notice one thing that needs improving now with more art pieces but that can be done later: https://github.com/Revolutionary-Games/Thrive/issues/3464

**Comment by Kasterisk** [comment]
The gallery music doesn't repeat, I think it would be better if it does, otherwise be in the gallery screen for long enough and there will just be silence.

**Review by Kasterisk — APPROVED** [review]
Regardless, looks good and thank you for these additions.

**Comment by hhyyrylainen** [comment]
> The gallery music doesn't repeat, I think it would be better if it does, otherwise be in the gallery screen for long enough and there will just be silence.

But it uses the jukebox? How can it not repeat...

**Comment by Kasterisk** [comment]
I don't know, it didn't repeat when I tested this locally earlier. Need someone else to confirm whether this is just me.

**Comment by hhyyrylainen** [comment]
If you pulled earlier, you need to reimport the sound file to make the loop property change work. If the tracks don't repeat with the jukebox, then the extinction theme looping and menu themes looping should also be broken.

**Pull request merged**